docker-for-mac/release-notes.md

```diff
@@ -64,14 +70,17 @@ about both kinds of releases, and download stable and edge product installers at
 
 ### Docker Community Edition 17.06.2-ce-mac27 2017-09-06 (Stable)
 
+[Download](https://download.docker.com/mac/stable/19124/Docker.dmg)
+
 * Upgrades
   - [Docker 17.06.2-ce](https://github.com/docker/docker-ce/releases/tag/v17.06.2-ce)
   - [Docker Machine 0.12.2](https://github.com/docker/machine/releases/tag/v0.12.2)
 
-### Docker Community Edition 17.06.1-ce-mac20, 2017-08-21 (stable)
+### Docker Community Edition 17.06.1-ce-mac24, 2017-08-21 (stable)
```

Comment:
why the build increment to 24? @ebriney Is this because we released build increments on 17.06.1 after `20` shipped?

Reply:
it is because edge releases, we increment the number on every mac release (from edge or stable)